lib/improver/tests/nowcasting/optical_flow/test_AdvectField.py

```diff
@@ -284,6 +284,14 @@ def test_validity_time(self):
         self.assertEqual(output_cube_time.hour, 4)
         self.assertEqual(output_cube_time.minute, 10)
 
+    def test_lead_time(self):
+        """Test output cube has a forecast_period coordinate with the correct
+        value and units"""
+        result = self.plugin.process(self.cube, self.timestep)
+        result.coord("forecast_period").convert_units("s")
+        lead_time, = result.coord("forecast_period").points
```

Comment:
This only checks the first point is the correct value and units not that there is only one point.